pages/agent/v3/cli_artifact.md.erb

```diff
@@ -198,7 +198,9 @@ There is no need to escape characters such as `?`, `[` and `]`.
 
 ## Downloading artifacts outside a running build
 
-The `buildkite-agent artifact download` command only works within the context of a running build.
+The `buildkite-agent artifact download` command only works within the context of running builds. It relies on environment variables that are set by the agent while a build is running. 
```

Comment:
I wonder if a clearer way to frame this might be "... command relies on environment variables that are set by the agent while a build is running."

Reply:
"within the context of running builds" seems confusing to me, and not actually accurate.

Reply:
Yeah, I like it! Added in 👌🏻